openwisp_users/tests/test_admin.py

```diff
@@ -169,6 +169,21 @@ def test_admin_change_user_email_empty(self):
         self.assertEqual(len(mail.outbox), 0)
         self.assertContains(response, 'errors field-email')
 
+    def test_admin_change_user_page_get_invalid_UUID(self):
+        admin = self._create_admin()
+        self.client.force_login(admin)
+        response = self.client.get(
+            reverse(f'admin:{self.app_label}_user_change', args=['WRONG'])
+        )
+        response2 = self.client.get(
+            reverse(
+                f'admin:{self.app_label}_user_change',
+                args=['15f0345b-b9c1-44b2-b674-0deac5c7c816'],
```

Comment:
```suggestion
                args=[uuid.uuid4()],
```
I believe instead of hard coding the identifier, using the `uuid` module to generate identifier will improve code readability.